lib/SILOptimizer/Mandatory/TFDifferentiation.cpp

```diff
@@ -3187,7 +3216,7 @@ class AdjointEmitter final : public SILInstructionVisitor<AdjointEmitter> {
   DenseMap<SILBasicBlock *, SILBasicBlock *> adjointBBMap;
 
   /// The range of original parameters in the adjoint function.
```

Comment:
It's no longer a "range of original parameters", since you changed it to a new array. Please change the comment to make it describe the new role of `originalParametersInAdj`.